src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -425,44 +432,62 @@ Environment variable: `$WEB_EXT_API_SECRET`
 
 #### `--api-url-prefix`
 
-The signing API URL prefix. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v4` which is the production API.
+The signing API URL prefix. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v4` which is the production API.  
+
+::: note alert
+This variable is not used with the add-on submission API. See --amo-base-url instead.
+:::
 
 Environment variable: `$WEB_EXT_API_URL_PREFIX`
 
+#### `--amo-base-url`
+
+The add-on submission API base URL. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v5` which is the production API.  
```

Comment:
```suggestion
The add-on submission API base URL. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v5` which is the production API.
```